### ISSUE TITLE
Warnings when running `bork run lint`/`bork run test`

### ISSUE THREAD

**Issue by duckinator** [duckinator/bork#176]
Getting some annoying/weird warnings that I've been unable to track down the cause of so far:

```
/usr/local/lib/python3.8/site-packages/setuptools/distutils_patch.py:25
  /usr/local/lib/python3.8/site-packages/setuptools/distutils_patch.py:25: UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
    warnings.warn(
```

<details>
<summary>Full logs</summary>

```
============================= test session starts ==============================
platform linux -- Python 3.8.5, pytest-6.0.1, py-1.9.0, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /tmp/cirrus-ci-build, configfile: pytest.ini
plugins: mypy-0.6.2, flake8-1.0.6, pylint-0.17.0
collecting ... collected 50 items
--------------------------------------------------------------------------------
Linting files
................
--------------------------------------------------------------------------------
setup.py::/tmp/cirrus-ci-build/setup.py::PYLINT PASSED                   [  1%]
setup.py::FLAKE8 PASSED                                                  [  3%]
setup.py::mypy PASSED                                                    [  5%]
bork/__init__.py::/tmp/cirrus-ci-build/bork/__init__.py::PYLINT PASSED   [  7%]
bork/__init__.py::FLAKE8 PASSED                                          [  9%]
bork/__init__.py::mypy PASSED                                            [ 11%]
bork/api.py::/tmp/cirrus-ci-build/bork/api.py::PYLINT PASSED             [ 13%]
bork/api.py::FLAKE8 PASSED                                               [ 15%]
bork/api.py::mypy PASSED                                                 [ 17%]
bork/asset_manager.py::/tmp/cirrus-ci-build/bork/asset_manager.py::PYLINT PASSED [ 19%]
bork/asset_manager.py::FLAKE8 PASSED                                     [ 21%]
bork/asset_manager.py::mypy PASSED                                       [ 23%]
bork/builder.py::/tmp/cirrus-ci-build/bork/builder.py::PYLINT PASSED     [ 25%]
bork/builder.py::FLAKE8 PASSED                                           [ 27%]
bork/builder.py::mypy PASSED                                             [ 29%]
bork/cli.py::/tmp/cirrus-ci-build/bork/cli.py::PYLINT PASSED             [ 31%]
bork/cli.py::FLAKE8 PASSED                                               [ 33%]
bork/cli.py::mypy PASSED                                                 [ 35%]
bork/filesystem.py::/tmp/cirrus-ci-build/bork/filesystem.py::PYLINT PASSED [ 37%]
bork/filesystem.py::FLAKE8 PASSED                                        [ 39%]
bork/filesystem.py::mypy PASSED                                          [ 41%]
bork/github.py::/tmp/cirrus-ci-build/bork/github.py::PYLINT PASSED       [ 43%]
bork/github.py::FLAKE8 PASSED                                            [ 45%]
bork/github.py::mypy PASSED                                              [ 47%]
bork/github_api.py::/tmp/cirrus-ci-build/bork/github_api.py::PYLINT PASSED [ 49%]
bork/github_api.py::FLAKE8 PASSED                                        [ 50%]
bork/github_api.py::mypy PASSED                                          [ 52%]
bork/log.py::/tmp/cirrus-ci-build/bork/log.py::PYLINT PASSED             [ 54%]
bork/log.py::FLAKE8 PASSED                                               [ 56%]
bork/log.py::mypy PASSED                                                 [ 58%]
bork/pypi.py::/tmp/cirrus-ci-build/bork/pypi.py::PYLINT PASSED           [ 60%]
bork/pypi.py::FLAKE8 PASSED                                              [ 62%]
bork/pypi.py::mypy PASSED                                                [ 64%]
bork/pypi_api.py::/tmp/cirrus-ci-build/bork/pypi_api.py::PYLINT PASSED   [ 66%]
bork/pypi_api.py::FLAKE8 PASSED                                          [ 68%]
bork/pypi_api.py::mypy PASSED                                            [ 70%]
bork/version.py::/tmp/cirrus-ci-build/bork/version.py::PYLINT PASSED     [ 72%]
bork/version.py::FLAKE8 PASSED                                           [ 74%]
bork/version.py::mypy PASSED                                             [ 76%]
tests/helpers.py::/tmp/cirrus-ci-build/tests/helpers.py::PYLINT PASSED   [ 78%]
tests/helpers.py::FLAKE8 PASSED                                          [ 80%]
tests/helpers.py::mypy PASSED                                            [ 82%]
tests/test_cmd_aliases.py::/tmp/cirrus-ci-build/tests/test_cmd_aliases.py::PYLINT PASSED [ 84%]
tests/test_cmd_aliases.py::FLAKE8 PASSED                                 [ 86%]
tests/test_cmd_aliases.py::mypy PASSED                                   [ 88%]
tests/test_cmd_aliases.py::test_cmd_aliases PASSED                       [ 90%]
tests/test_cmd_download.py::/tmp/cirrus-ci-build/tests/test_cmd_download.py::PYLINT PASSED [ 92%]
tests/test_cmd_download.py::FLAKE8 PASSED                                [ 94%]
tests/test_cmd_download.py::mypy PASSED                                  [ 96%]
tests/test_cmd_download.py::test_download PASSED                         [ 98%]
::mypy PASSED                                                            [100%]
=============================== warnings summary ===============================
/usr/local/lib/python3.8/site-packages/setuptools/distutils_patch.py:25
  /usr/local/lib/python3.8/site-packages/setuptools/distutils_patch.py:25: UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
    warnings.warn(
-- Docs: https://docs.pytest.org/en/stable/warnings.html
===================================== mypy =====================================
Success: no issues found in 16 source files
======================== 51 passed, 1 warning in 11.95s ========================
/usr/local/lib/python3.8/site-packages/setuptools/distutils_patch.py:25: UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
  warnings.warn(
```

</details>

**Comment by duckinator** [comment]
Turns out `bork.github._get_release_info(...)` uses `distutils.version.LooseVersion`. This can possibly be replaced by something in the `packaging` library.

**Comment by duckinator** [comment]
This should be fixed by #180.
